Yoast/wordpress-seo
# Use the `wp_doing_ajax` function instead of constants

## Summary

This PR can be summarized in the following changelog entry:

* Uses the wp_doing_ajax functions instead of doing constant checks

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:

* There is no functional change, because I used the wrapper from WordPress
* You can verify this code by checking if AJAX requests are still executed.

## UI changes

## Quality assurance


Related #9570
